src/cli/index.ts

```diff
@@ -30,11 +31,35 @@ async function cli(args: string[]) {
     case 'verify':
       await verify(args[1], args[2]);
       break;
+    case 'version':
+    case '-version':
+    case '--version':
+    case '-v':
+      console.log(packageJson.version);
+    console.log("bajja");
```

Comment:
I'm guessing this is something leftover from debugging?

Reply:
Sorry about that, I'll remove it shortly. Thanks